CONTRIBUTING.md

```diff
@@ -7,6 +7,10 @@ Please read through this document before submitting any issues or pull requests
 information to effectively respond to your bug report or contribution.
 
 
+## Security issue notifications
+If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public github issue.
+
+
```

Comment:
I moved this to a) match better other such files and b) because I had to take the anchor out of the link in README.rst because sphinx couldn't see it.

Reply:
nit: I've always thought this should be in the README outright,  
but that's a personal view.